docs/cluster-configuration.md

```diff
@@ -166,6 +166,9 @@ For additional information, see here: [KubeSecondaryDNS](https://github.com/kube
 
 **Default**: `false`
 
+In order to set KSD's NameServerIP, set it on HyperConverged CR spec.KubeSecondaryDNSNameServerIP field.
+Default: empty string. Value is a string representation of IPv4 (i.e "127.0.0.1").
+
```

Comment:
We have section for each field. I understand that the `deployKubeSecondaryDNS` FG and the new `NameServerIP` are connected, and that there is no point to write the same documentation twice, but we could add a new section for the `NameServerIP` with a reference to this section.

Also, still missing an example (and there is also no example to the `deployKubeSecondaryDNS` FG).